### ISSUE TITLE
Title attribute

### ISSUE THREAD

**Issue by trf000** [AppStateESS/phpwebsite#274]
The title attribute in Menu, Read More in blog and set in image upload in Filecabinet and CKEditor should be removed. Title is not used for images, and actually hinders accessibility for links.

for blog, Blog.php line 399 if you change:
$template['READ_MORE'] = PHPWS_Text::rewriteLink(Icon::get('chevron-circle-down') . '&nbsp;' . dgettext('blog', 'Read more'), 'blog',

to something like

$template['READ_MORE'] = PHPWS_Text::rewriteLink(Icon::get('chevron-circle-down') . '&nbsp;' . dgettext('blog', 'Read more <span class="sr-only">of '.$this->title.'</span>'), 'blog',
you'll get a more meaningful link for the visually impaired.
